crates/stack/src/connection.rs

```diff
@@ -6,9 +6,9 @@ use std::pin::Pin;
 use std::rc::Rc;
 use std::task::{Context, Poll};
 
-use smoltcp::iface::SocketHandle;
-use smoltcp::socket::*;
-use smoltcp::wire::IpEndpoint;
+use ya_smoltcp::iface::SocketHandle;
```

Comment:
Just hint for future: we could rename `ya_smoltcp` as `smoltcp` to avoid most of the changes.
I remember that maybe this is even possible on cargo.toml level

Reply:
yes, you can add in Cargo.toml:
`smoltcp = { package = "ya-smoltcp", version = "0.1" }`, but I have not seen that anywhere in the current Golem Factory codebase, nor I have not ever use this approach